Sources/MongoSwift/MongoCollection.swift

```diff
@@ -12,6 +12,33 @@ public struct DropCollectionOptions: Codable {
     }
 }
 
+/// Options to use when renaming a collection.
+public struct RenameCollectionOptions: Codable {
```

Comment:
I see that `DropCollectionOptions` were also included in this file, but I think that was a mistake. We generally include the options for a given operation in the operations file. So in this case we'd have these options in `RenameCollectionOperation.swift`.